net-analyzer/ipsumdump/ipsumdump-1.85.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Simple TCP/IP Dump summarizer/analyzer"
+HOMEPAGE="http://read.seas.harvard.edu/~kohler/ipsumdump/"
+SRC_URI="http://read.seas.harvard.edu/~kohler/ipsumdump/ipsumdump-1.85.tar.gz"
+
+LICENSE="the-Click-license"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+ipv6 +nanotimestamp"
+
+DEPEND="dev-lang/perl
+	dev-libs/expat
+	net-libs/libpcap
+	sys-apps/texinfo"
+RDEPEND="${DEPEND}"
+
+WANT_AUTOCONF=2.60
+
+src_prepare() {
+	eapply_user
```

Comment:
By default phase I mean what will happen if you don't define a custom src_prepare.

In EAPI 6, if you do not declare your own src_prepare the [default](https://projects.gentoo.org/pms/latest/pms.html#x1-990009.1.5) is run which applies $PATCHES and calls eapply_user.